scripts/azure-pipelines.yml

```diff
@@ -7,50 +7,64 @@ jobs:
 - job: BuildAndTest
   timeoutInMinutes: 360
 
+  strategy:

```

Comment:
If anyone knows how to do this in a nicer way, let me know. E.g. it's a lot of copy/paste to add a new "value" and a LOT more copy paste to add a new "variable".

Reply:
Can you do a `matrix` like cppwinrt?  https://github.com/microsoft/cppwinrt/blob/2511bf7fcb54a64c1e596a950788e2ee30cd1e4c/.pipelines/build.yml#L29

Reply:
It appears that matrix supports nesting: https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/jobs-job-strategy?view=azure-pipelines

Reply:
Missed the comment about `each` on my first skim through. Not too happy with the `split`, however the docs say that variables must be strings, so arrays don't work :/